R/aggreger_ki_prop.R

```diff
@@ -118,21 +118,23 @@ aggreger_ki_prop = function(d_ki_ind, alfa = 0.05) {
     ungroup()
 
   # Legg til konfidensintervall
-  konfint_robust = function(x) {
-    konf = possibly(~ regn_konfint_bin(d_sammendrag$ki_teller,
-      d_sammendrag$ki_nevner,
-      alfa
-    ),
-    otherwise = data.frame(
-      method = NA_character_,
-      x = NA_integer_,
-      n = NA_integer_,
-      mean = NA_real_,
-      lower = NA_real_,
-      upper = NA_real_
+  konfint_robust = function() {
```

Comment:
Kva skjedde med `x`-argumentet?

Reply:
Det var eit argument som ikkje var i bruk?

Reply:
Men burde ha fjerna det i ein eigen commit. Kan fiksa det.

Reply:
Hm. Det stemmer visst. Men er ikkje det ganske ufint? Kunne jo tatt inn `d_sammendrag`, slik at funksjonen vert lettare å forstå. Men det er fleire rare ting her. For eksempel at ein brukar ein anonym funksjon som brukar ein ekstern variabel.

Men du kan la det vera, og så rettar me når me ser nøyare på heile funksjonen (det trengst).